OpenKeychain/src/main/res/xml/sync_preferences.xml

```diff
@@ -4,6 +4,11 @@
         android:persistent="false"
         android:title="@string/label_sync_settings_keyserver_title"/>
     <SwitchPreference
+        android:key="enableWifiSyncOnly"
+        android:persistent="true"
+        android:dependency="syncKeyserver"
```

Comment:
Alright :)
So I guess everything checks out, could you go ahead and merge this?